src/databricks/labs/ucx/cli.py

```diff
@@ -398,5 +398,21 @@ def assign_metastore(
     ctx.account_metastores.assign_metastore(ctx.prompts, workspace_id, metastore_id, default_catalog)
 
 
+@ucx.command
+def migrate_tables(w: WorkspaceClient):
+    """Trigger the migrate-tables workflow"""
+    ctx = WorkspaceContext(w)
+    deployed_workflows = ctx.deployed_workflows
+    deployed_workflows.run_workflow("migrate-tables")
+
+
+@ucx.command
+def migrate_external_hiveserde_tables_in_place(w: WorkspaceClient):
```

Comment:
Can we merge this into one command and prompt user to trigger hiveserde job only if there are such tables where we show some numbers? Also, how should we deal with CTAS? We have to prompt user for a choice.